react/wbooks/src/app/components/PublicRoute/index.js

```diff
@@ -7,7 +7,7 @@ function PublicRoute({ component: Component, redirect, ...rest }) {
     <Route
       {...rest}
       render={props =>
-        localStorage.getItem('token_id') ? <Redirect to={redirect} /> : <Component {...props} />
+        localStorage.getItem('tokenId') ? <Redirect to={redirect} /> : <Component {...props} />
```

Comment:
You can use only 1 component `Route` and pass a prop `private`. If it is private you negate this condition. And you end up using the same component for the two routes